packages/augur-ui/src/modules/market/components/market-view/market-view.tsx

```diff
@@ -260,7 +262,8 @@ export default class MarketView extends Component<MarketViewProps, MarketViewSta
       );
     }
 
-    const outcome = outcomes.find( outcomeValue => outcomeValue.id === s.selectedOutcomeId)
+    const outcomeId = s.selectedOutcomeId ? s.selectedOutcomeId : market.defaultSelectedOutcomeId;
```

Comment:
if market is lazy loaded need to set default selected outcome id.

Reply:
👍